index.d.ts

```diff
@@ -142,6 +157,7 @@ export function parse(query: string, options?: ParseOptions): ParsedQuery;
 export interface ParsedUrl {
 	readonly url: string;
 	readonly query: ParsedQuery;
+	readonly fragmentIdentifier?: string;
```

Comment:
This one is not documented in the readme.

Reply:
Which section would this go under?

Reply:
I feel like that's quite obvious: https://github.com/sindresorhus/query-string/pull/222/files#diff-0730bb7c2e8f9ea2438b52e419dd86c9R344